tensorflow/java/src/main/java/org/tensorflow/Operation.java

```diff
@@ -91,6 +91,21 @@ public int outputListLength(final String name) {
     }
   }
 
+  /**
+   * Returns symbolic handles to a list of tensors produced by this operation.
+   *
+   * @param idx index of the first tensor of the list
+   * @param length number of tensors in the list
+   * @return array of {@code Output}
+   */
+  public Output[] outputList(int idx, int length) {
```

Comment:
Ok good. Just to explain my thoughts around this a little more, I agree that `outputList(String name)` is a most intuitive API for getting hold on an output list but in fact, since we collect iteratively the outputs of an operation, the use of the index is more useful. For example, if you have three outputs where the last one is a list, you automatically know that it starts from index 3, and so on.